environment.yml

```diff
@@ -2,6 +2,8 @@ name: pubtator
 dependencies:
 - python=3.6.0
 - lxml=3.7.2
+- tqdm=4.11.2
+- pandas=0.20.2
 - pip:
     - git+https://github.com/en-dash/PyBioC@a21bd181ce3b89de791db4bb7db6ca1c48167648
-    - tqdm==4.11.2
+    
```

Comment:
The problem here (why github shows the red symbol) is that the final newline has some whitespace on it. Instead it should be blank.